bindings/node/test/types/index.test-d.ts

```diff
@@ -0,0 +1,30 @@
+import { expectAssignable, expectError } from 'tsd';
+import { ClientEncryption } from '../..';
+
+type RequiredCreateEncryptedCollectionSettings = Parameters<
+  ClientEncryption['createEncryptedCollection']
+>[2];
+
+expectError<RequiredCreateEncryptedCollectionSettings>({});
```

Comment:
do we have precedence for type tests like this?  It seems like all that's being tested is optionality of arguments.

Reply:
I made the typescript change the type of CreateCollectionOptions to be the same except `encryptedFields` is now required, so I wanted to demonstrate that in a test so it couldn't be changed without notice down the road